planning/scenario_planning/common/motion_velocity_optimizer/scripts/trajectory_visualizer.py

```diff
@@ -572,8 +601,10 @@ def updatePose(self, from_link, to_link):
             print("updatePose succeeded")
```

Comment:
I think it's better to use `self.get_logger.info()` instead of `print()`.

Reply:
Thanks! Fixed at e208dc6.